tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d

```diff
@@ -12,8 +12,8 @@ export SMONTH
 export SDAY="01"
 export SHOUR="00"
 
-export DAYS="35"
-export FHMAX="840"
+export DAYS="2"
```

Comment:
Should this be put back to 35 and 840?

Reply:
@DeniseWorthen Should we change this back to 35 days?

Reply:
Yes before we commit, we need to change this back. It was only used to be able to create a short test vs running the whole 35d.

I want to do one more test w/ my cice6 branch, which is to run more than 1 date.  I expect it to work. 

I haven't done the ww3 35d test though. Did you test that Minsuk?

Reply:
I have run 35 day (well, 48 hours) tests for all dates with and without ww3.

Reply:
OK, great. It worked for more than one day in my cice6 branch too so I think it is good.